File: rb/test/exists_test.rb
# Coingecko SDK exists test

require "minitest/autorun"
require_relative "../Coingecko_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = CoingeckoSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- Coingecko SDK exists test

local sdk = require("coingecko_sdk")

describe("CoingeckoSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
